### PR TITLE
[WIP] Add sidebar.

### PR DESCRIPTION
**Pull request by franekp**

### CODE REVIEW

File: timesketch/lib/experimental/create_graph.cql [hunk 1]
```diff
@@ -76,7 +76,7 @@ WITH line
 
   MERGE (src:WindowsMachine { hostname: line.src, sketch_id: {sketch_id} })
   MERGE (service:WindowsService { service_name: line.svc_name, sketch_id: {sketch_id} })
-  MERGE (image_path:WindowsServiceImagePath { image_path: line.image_path, sketch_id: {sketch_id} })
+  MERGE (image_path:WindowsServiceImagePath { image_path: line.image_path, image_path_short: line.image_path_short, sketch_id: {sketch_id} })
```
Comment (berggren) [inline timesketch/lib/experimental/create_graph.cql]: Do we need this in the MERGE, wouldn't line.image_path be enough end then just set image_path_short in the SET?

File: timesketch/ui/graphs/graph-view.data.ts [hunk 2]
```diff
@@ -59,45 +59,56 @@ export const style = ({nodeLabel, edgeLabel}): CytoscapeStyle => [
...
       'text-halign': 'center',
       'text-valign': 'center',
       'color': '#FFFFFF',
       'font-size': '10',
-      'background-color': '#68BDF6',
-      'border-color': '#5CA8DB',
-      'border-width': '1',
+      'font-weight': 'bold',
+      'text-outline-width': '3px',
+      'padding': '3px',
```
Comment (berggren) [inline timesketch/ui/graphs/graph-view.data.ts]: I would like to see how 5px would look like here. Just a minor thing though :)

File: timesketch/lib/experimental/create_graph.cql (hunk 1 above)
Comment (franekp) [reply timesketch/lib/experimental/create_graph.cql]: Ok. I will fix it in one of next commits.

File: timesketch/ui/graphs/graph-view.data.ts (hunk 2 above)
Comment (franekp) [reply timesketch/ui/graphs/graph-view.data.ts]: 3px
4px
5px

File: timesketch/lib/experimental/create_graph.cql (hunk 1 above)
Comment (berggren) [reply timesketch/lib/experimental/create_graph.cql]: ack

File: timesketch/ui/graphs/graph-view.data.ts (hunk 2 above)
Comment (franekp) [reply timesketch/ui/graphs/graph-view.data.ts]: 3, 4 and 5 px respectively. I changed it to 4.

File: timesketch/lib/experimental/create_graph.cql (hunk 1 above)
Comment (franekp) [reply timesketch/lib/experimental/create_graph.cql]: Actually, putting it in SET is avoids one bug.

File: timesketch/lib/datastores/neo4j.py [hunk 3]
```diff
@@ -16,34 +16,46 @@
...
+HIDDEN_FIELDS = [
+    'type', 'id', 'sketch_id', 'source', 'target', 'es_index_name', 'es_query',
+]
+
 # Schema for Neo4j nodes and edges
 SCHEMA = {
     u'nodes': {
         u'WindowsMachine': {
-            u'label_template': u'\uf009  {hostname}',
+            u'label': u'\uf009\u00A0\u00A0{hostname}',
```
Comment (berggren) [inline timesketch/lib/datastores/neo4j.py]: Maybe use uf17a (the Windows logo) instead?

Comment (franekp) [reply timesketch/lib/datastores/neo4j.py]: No. Windows logo does not look good when in bold font.

**Pull request merged**